pkg/codec/splunkhec/splunkhec_to_metricdata.go

```diff
@@ -78,32 +76,28 @@ func splunkHecToMetricsData(logger *zap.Logger, events []*splunk.Event, resource
 
 			attrs := resourceMetrics.Resource().Attributes()
 			if event.Host != "" {
-				attrs.PutStr(config.HecToOtelAttrs.Host, event.Host)
+				attrs.PutStr(config.Host, event.Host)
 			}
 			if event.Source != "" {
-				attrs.PutStr(config.HecToOtelAttrs.Source, event.Source)
+				attrs.PutStr(config.Source, event.Source)
 			}
 			if event.SourceType != "" {
-				attrs.PutStr(config.HecToOtelAttrs.SourceType, event.SourceType)
+				attrs.PutStr(config.SourceType, event.SourceType)
 			}
 			if event.Index != "" {
-				attrs.PutStr(config.HecToOtelAttrs.Index, event.Index)
-			}
-			if resourceCustomizer != nil {
-				resourceCustomizer(resourceMetrics.Resource())
+				attrs.PutStr(config.Index, event.Index)
 			}
 		}
 		metrics.MoveAndAppendTo(sm.Metrics())
 	}
 
-	return md, numDroppedTimeSeries
+	return md
 }
 
-func convertString(logger *zap.Logger, numDroppedTimeSeries *int, metrics pmetric.MetricSlice, metricName string, pointTimestamp pcommon.Timestamp, s string, attributes pcommon.Map) {
+func convertString(logger *zap.Logger, metrics pmetric.MetricSlice, metricName string, pointTimestamp pcommon.Timestamp, s string, attributes pcommon.Map) {
 	// best effort, cast to string and turn into a number
 	dbl, err := strconv.ParseFloat(s, 64)
 	if err != nil {
-		*numDroppedTimeSeries++
```

Comment:
This and the change in line 99 are just cleanup, right?

Reply:
Correct. This counter was only used in testing, and didn't really do well with the Marshaler interface. There was no point in keeping it.